public/app/features/alerting/unified/components/MoreButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef, Ref } from 'react';
+
+import { Stack } from '@grafana/experimental';
+import { Button, ButtonProps, Icon } from '@grafana/ui';
+
+const MoreButton = forwardRef(function MoreButton(props: ButtonProps, ref: Ref<HTMLButtonElement>) {
```

Comment:
We can use this one in several other places, provides us with some consistency for how to render "more" buttons.

Currently hard-coded to `sm` button size but can be overridden.

Reply:
I'm using this component as a child component in `Menu.Item` which seems to forward refs to the child components – we need to wrap it here with `forwardRef` to make sure we keep the original ref that was passed